src/Network/Network/AzureFirewall/NewAzureFirewallCommand.cs

```diff
@@ -54,22 +55,56 @@ public class NewAzureFirewallCommand : AzureFirewallBaseCmdlet
         [ValidateNotNullOrEmpty]
         public virtual string Location { get; set; }
 
+        [CmdletParameterBreakingChange(
+            "VirtualNetworkName",
+            deprecateByVersion: "2.0.0",
+            ChangeDescription = "This parameter will be removed in an upcoming breaking change release. After this point the Virtual Network will be provided as an object instead of a string.", 
+            OldWay = "New-AzureRmFirewall -VirtualNetworkName \"vnet-name\"",
+            NewWay = "New-AzureRmFirewall -VirtualNetwork $vnet",
```

Comment:
`OldWay` and `NewWay` should call `New-AzFirewall`, not AzureRM.

Reply:
Done, thank you.